### DISCUSSION TITLE
v8 - disable sorting for a specfic column

### DISCUSSION THREAD

**Discussion by chiptus** [TanStack/table#3880] [Q&A] [answered]
I don't see an example in https://github.com/TanStack/react-table/blob/alpha/examples/sorting/src/main.tsx how to disable sorting per columns

**Comment by tannerlinsley** [comment, marked as answer]
Set `columnDef.enableSorting = false`

**Reply by chiptus** [reply]
thanks!

**Comment by lamhungypl** [comment]
How can I enable sorting for specific columns instead of iterating all other columDef and `set columnDef.enableSorting = false`. 
suppose there is some API can set default value for `columnDef.enableSorting`

**Reply by chiptus** [reply]
use https://tanstack.com/table/v8/docs/api/core/table#defaultcolumn

```js
useTable({
  defaultColumn: {
     enableSorting: false
  }
})
```
